blitz/cli/commands/start.py

```diff
@@ -73,7 +73,7 @@ def start_blitz(
             host="localhost",
             port=port,
             reload=True,
-            reload_includes=str(blitz_app.file.path.relative_to(Path.cwd())),
+            reload_includes=[str(blitz_app.file.path.relative_to(Path.cwd())), "blitz/api/templates"],
```

Comment:
only for dev purpose no ? do we really want to include that in the main code ?

Reply:
Yes and that doesn't work